modules/ossm-integrating-with-user-workload-monitoring.adoc

```diff
@@ -224,7 +227,9 @@ spec:
 ====
 If there is only one mesh using user-workload monitoring, then both the `mesh_id` relabeling and the `spec.prometheus.query_scope` field in the Kiali resource are optional (but the `query_scope` field given here should be removed if the `mesh_id` label is removed).
 
-If there might be multiple meshes using user-workload monitoring, then both the `mesh_id` relabelings and the `spec.prometheus.query_scope` field in the Kiali resource are required so that Kiali only sees metrics from its associated mesh. If Kiali is not being deployed, applying the `mesh_id` relabeling is still recommended so that metrics from different meshes can be distinguished from one another.
+If multiple mesh instances on the cluster may use user-workload monitoring, then both the `mesh_id` relabelings and the `spec.prometheus.query_scope` field in the Kiali resource are required. This ensures that Kiali only sees metrics from its associated mesh.
```

Comment:
I will not hold up merging, but IBM Style doesn't like "may" - https://www.ibm.com/docs/en/ibm-style?topic=word-usage#m

Reply:
Ah, thanks for catching that! I have another PR open with this same file so I will go make that change in that PR. Thank you!